docs/content/latest/explore/ysql-language-features/gin.md

```diff
@@ -0,0 +1,501 @@
+# Generalized inverted indexes
+
+In YugabyteDB, tables and secondary indexes are both key-value stores.
+For tables, the key-value store maps primary keys to values.
+For secondary indexes, the key-value store maps index keys to primary keys.
+
+**Regular indexes index columns.**
+This makes queries with conditions on the columns more efficient.
+For example, if one had a regular index on a single int array column (currently not possible in Yugabyte), queries like `WHERE myintarray = '{1,3,6}'` would be more efficient when using the index.
```

Comment:
`not possible in Yugabyte` -- not that in matters as we don't support it in YCQL either, but maybe `in YSQL` is clearer.